pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/db/dao/AnswerDao.java

```diff
@@ -520,7 +520,7 @@ public Answer reduce(Map<Long, Answer> container, RowView view) {
                                 view.getColumn("fa_file_name", String.class),
                                 view.getColumn("fa_file_size", Long.class));
                     }
-                    ((FileAnswer) answer).getValue().add(info);
+                    ((FileAnswer) answer).getValue().add(info); //todo : check adding null FileInfo if no FileAnswer ?
```

Comment:
There is a testcase testPatch_fileAnswer_canCreateNullValue_andClearValueBySendingNull in PatchFormAnswersRouteStandaloneTest which made me move away from making the fix here and handling it in DataExporter.
Added a todo to revisit and check back later

Reply:
@LKapanad You were the last to modify that test- Can you explain what is going on in [PatchFormAnswersRouteStandaloneTest.java:1920](https://github.com/broadinstitute/ddp-study-server/blob/6f1488319fb997a9cb9d1c6c12b5a55a8bfadac5/pepper-apis/dss-core/src/test/java/org/broadinstitute/ddp/route/PatchFormAnswersRouteStandaloneTest.java#L1920)? That line should be receiving a `java.lang.IndexOutOfBoundsException`, but it appears we're inserting `null` values into the array instead.

Reply:
updated unit test and fixed the bug